site/en/models.md

```diff
@@ -41,6 +41,17 @@ using `ocrd_all`:
   * [script/Fraktur](https://github.com/tesseract-ocr/tessdata_fast/raw/master/script/Fraktur.traineddata)
   * [@stweil's GT4HistOCR model](https://ub-backup.bib.uni-mannheim.de/~stweil/ocrd-train/data/fast/Fraktur_50000000.334_450937.traineddata)
 
+If you installed Tesseract with Ubuntu's `apt` package manager, you may want to install
+standard models like `deu` or `script/Fraktur` with `apt`:
+
+```sh
+sudo apt install tesseract-ocr-deu tesseract-ocr-script-frak
```

Comment:
Oh, but that's already been covered automatically in ocrd_all's Makefile. It downloads models via `wget` when building Tesseract from source, but tries to use the (differently named) deb packages when installing Tesseract from PPA. You only specify the model names (without suffix) in `TESSERACT_MODELS` variable in both cases...

Reply:
My assumption here is an installation of tesseract outside of ocrd_all (e.g.using Ubuntu 19.10+). Should we clarify that we recommend to use alex-p's PPA and change the package names accordingly?

Reply:
I don't know, depends on the overall reading flow of the guide I guess. It's up to you

Reply:
OK then let's hope the subtle references to ocrd_all will guide readers in the right direction and keep the Ubuntu notes for those that set up OCR-D on their own.